src/release/breaking-changes/wide-gamut-framework.md

```diff
@@ -0,0 +1,135 @@
+colors.
+
+## Context
+
+The Flutter engine [already supports wide gamut color][] with [Impeller][], and the support is now being added [to the framework][].
+
+The iOS devices Flutter targets support rendering a larger array of colors,
+specifically the [DisplayP3][] color space. After this change, the Flutter
+framework will be able to render all of those colors on iOS Impeller and `Color`
+will be better prepared for the future addition of color spaces or changes to
+color component bit depth.
+
+## Description of change
+
+Changes to [`Color`][]:
+
+ 1. Adds a enum field which will specify its [`ColorSpace`][].
+ 1. Adds API to use normalized floating-point color components.
+ 1. Removes API that uses 8-bit unsigned integer color components that can lead
+    to data loss.
+
+Changes to [`ColorSpace`][]:
+
+ 1. `displayP3` will be added.
+
+## Migration guide
+
+### 8-bit unsigned integer constructors
+
+Constructors like `Color.fromARGB` will remain unchanged and have continued
+support. If one wants to take advantage of Display P3 colors they will have to
+use the new `Color.from` constructor that takes normalized floating-point
+color components.
+
+```dart
+// Before
+final magenta = Color.fromARGB(0xff, 0xff, 0x0, 0xff);
+// After
+final magenta = Color.from(alpha: 1.0, red: 1.0, green: 0.0, blue: 1.0)
+```
+
+### Implementors of `Color`
+
+There are new methods being added to `Color` so any class that `implements`
+Color will break and have to implement the new methods like `Color.a`, and
+`Color.b`. Ultimately implementors should be migrated to take advantage of the
+new API. In the short-term, these methods can easily be implemented without
+changing the underlying structure of your class.
+
+For example:
+
+```dart
+class Foo implements Color {
+  int _red;
+
+  @override
+  double get r => _red * 255.0;
+}
+```
+
+{{site.alert.note}}
+  The hope is to eventually make Color `sealed`. Now might be a good opportunity
+  to stop implementing `Color`. Instead of inheritance, switch to composition.
+{{site.alert.end}}
+
+### Color space support
+
+Clients that use `Color` and perform any sort of calculation on the color
+components should now be checking the color space component before doing any
+such calculation. `Color` has a new method `Color.withValues` that can be used
+to perform color space conversions to help with that.
+
+For example:
+
+```dart
+// Before
+double redRatio(Color x, Color y) => x.red / y.red;
+
+// After
+double redRatio(Color x, Color y) {
+  final xPrime = x.withValues(colorSpace: ColorSpace.extendedSRGB);
+  final yPrime = y.withValues(colorSpace: ColorSpace.extendedSRGB);
+  return xPrime.r / yPrime.r;
+}
+```
+
+### Color component accessors
+
+For users of `Color`, if the components are ever accessed the code should be
+migrated to take advantage of the floating-point components. In the short term
+one can easily scale the components themselves.
```

Comment:
"In the short term"? This feels unfinished. What happens in the long term? 

```suggestion
If your app ever accesses a `Color` component, consider taking
advantage of the floating-point components. In the short term,
you can easily scale the components themselves.
```

Reply:
The long term is the first phrase.  That's the preferred course of action.  The short term suggestion is there if for some reason that's not an easy change.

Reply:
I still prefer the suggestion.  :)

Reply:
sgtm, done